ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -76,6 +76,7 @@ class MSFButtonStateImpl: NSObject, ObservableObject, ControlConfiguration, MSFB
     var action: () -> Void
     @Published var image: UIImage?
     @Published var disabled: Bool?
+    @Published var focused: Bool?
```

Comment:
Can this be of type Bool and have a default to false?

`@Published var focused: Bool = false`

Then we could discard the computed property (as we're not exposing them in the protocol for ObjC consumption).